ZipUtilities/NOZSyncStepOperation.m

```diff
@@ -179,23 +179,26 @@ - (void)updateProgress:(float)progress forStep:(NSUInteger)step
 
     const float oldProgress = self.progress;
     const BOOL wasIndeterminate = oldProgress < 0.f;
-    BOOL isIndeeterminate = progress < 0.f;
+    BOOL isIndeterminate = wasIndeterminate || (progress < 0.f);
```

Comment:
“Is” not based on “was”.  Please leave the logic as it was

Reply:
in the old logic, if `was`  is true, then `is` may be calculate, but it will never be read, because the `was` check on line 184 will cause the `is` check on that line to never occur.

in the old logic, if `was` is false, only then will `is` have any meaning.

so, `is` is, tautologically, based on `was` in this case.

that is the same as the new logic … except the `(progress < 0.f)` will never needlessly be calculated in the new logic as it was in the old logic.